pkg/auth/mock.go

```diff
@@ -0,0 +1,16 @@
+package auth
```

Comment:
Usually, mocks are in a separate package beside the main implementation. Mainly to keep the package api clean from testing code.